Doc/library/pprint.rst

```diff
@@ -44,6 +44,17 @@ Functions
    *args* and *kwargs* will be passed to :func:`~pprint.pprint` as formatting
```

Comment:
Should this reference also be changed?

```suggestion
   *args* and *kwargs* will be passed to :func:`~pprint.pp` as formatting
```

Reply:
Nah, it's correct, because it's a code description.
Lib\pprint.py:64
```py
def pp(object, *args, sort_dicts=False, **kwargs):
    """Pretty-print a Python object"""
    pprint(object, *args, sort_dicts=sort_dicts, **kwargs)
```

Reply:
Ok, thanks!